src/gui/qgsmetadatawidget.h

```diff
@@ -43,16 +43,50 @@ class GUI_EXPORT QgsMetadataWidget : public QWidget, private Ui::QgsMetadataWidg
 
     /**
      * Constructor for the wizard.
+     * \note
+     * For use with Layer. Sets mMetadata if pointer is valid
+     *  calls setMetadata, using mMetadata
+     * \param layer to set the main QgsLayerMetadata with mLayer->metadata() when not nullptr
+     * \see mMetadata
+     * \see setMetadata
+     * \since QGIS 3.0
      */
     QgsMetadataWidget( QWidget *parent, QgsMapLayer *layer = nullptr );
 
+    /**
+     * Set a QgsLayerMetadata object.
+     * \note
+     * Called from constructor and initializes child widged on first use
+     * Can be called from outside to change the QgsLayerMetadata object.
+     * \param layerMetadata to set the main  QgsLayerMetadata
+     * \see mMetadata
```

Comment:
This should be "\see metadata()". We don't want private variables documented in the public docs like this.

Reply:
done